### ISSUE TITLE
2.7.23 incompatible with OGSpy Xtense module 2.7.4? 

### ISSUE THREAD

**Issue by juliogonzalez** [OGSteam/tool-xtense-web-extension#30]
OGSpy version: 3.3.4
OGSpy module version: 2.7.4 (latest stable)
Web extension version: 2.7.23

And the error is:
![image](https://user-images.githubusercontent.com/4226070/55674816-d65b1c00-58b9-11e9-94cb-ebeabc087f15.png)

At the web server logs I can't see anything strange. At any time when the browser module is supposed to upload data, I see:

> 172.17.0.1 localhost:8008 - [06/Apr/2019:20:26:53 +0000] "POST /mod/xtense/xtense.php HTTP/1.1" 200 36 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36"

In case it's relevant: OGSpy is hosted on a docker container exposed with a reverse proxy. Should not affect, but...

**Comment by darknoon29** [comment]
Hello, what is your ogame url (to know your country) ? Do you have more logs in the web browser console ? (By enabling Debug mode in the mod settings ? )

**Comment by darknoon29** [comment]
Concerning the question it should be compatible :-) I tested it with at least French language ;-)

**Comment by juliogonzalez** [comment]
Ogame URL: https://s159-es.ogame.gameforge.com/

From the Web console:
```
[1554582714261] Xtense : Universe Number: 159
utilities.js:63 [1554582714261] Xtense : Universe language: es
utilities.js:63 [1554582714269] Xtense : setStatus : Xtense Ready
utilities.js:63 [1554582714270] Xtense : Static Observer : Running system
utilities.js:63 [1554582714271] Xtense : In parse_galaxy_system_inserted()
utilities.js:63 [1554582714371] Xtense : Mutation Observer : [object NodeList]
utilities.js:63 [1554582714372] Xtense : In parse_galaxy_system_inserted()
utilities.js:63 [1554582714374] Xtense : setStatus : Solar system detected(4:394)
utilities.js:63 [1554582714374] Xtense : 19 rows found in galaxy
utilities.js:63 [1554582714375] Xtense : row 1 has no player name
utilities.js:63 [1554582714375] Xtense : row 2 has no player name
utilities.js:63 [1554582714375] Xtense : row 3 has no player name
utilities.js:63 [1554582714376] Xtense : row 4 has no player name
utilities.js:63 [1554582714376] Xtense : row 5 has no player name
utilities.js:63 [1554582714376] Xtense : row 6 has no player name
utilities.js:63 [1554582714376] Xtense : row 7 has no player name
utilities.js:63 [1554582714377] Xtense : row 8 has no player name
utilities.js:63 [1554582714377] Xtense : row 9 has no player name
utilities.js:63 [1554582714377] Xtense : row 10 has no player name
utilities.js:63 [1554582714378] Xtense : row 11 has no player name
utilities.js:63 [1554582714379] Xtense : Ally id500475
utilities.js:63 [1554582714379] Xtense : row 8 > player_id:107688,planet_name:Rostovia,planet_id:33687810,moon_id:,moon:0,player_name:Leonov,status:,ally_id:500475,ally_tag:Odys...,debris:(0/0),activity:0,activity_moon:-1
utilities.js:63 [1554582714380] Xtense : row 13 has no player name
utilities.js:63 [1554582714381] Xtense : row 14 has no player name
utilities.js:63 [1554582714382] Xtense : row 15 has no player name
utilities.js:63 [1554582714382] Xtense : row 16 has no player name
utilities.js:63 [1554582714383] Xtense : row 17 has no player name
utilities.js:63 [1554582714383] Xtense : row 18 has no player name
utilities.js:63 [1554582714384] Xtense : row 19 has no player name
utilities.js:63 [1554582714388] Xtense : sending toolbar_version=2.7.23&toolbar_type=GM-GC&mod_min_version=2.7.4&user=admin&password=70ecb1fe650f5c7dcbdd8c4d62130234&univers=https://s159-es.ogame.gameforge.com&row[8][player_id]=107688&row[8][planet_name]=Rostovia&row[8][planet_id]=33687810&row[8][moon_id]=&row[8][moon]=0&row[8][player_name]=Leonov&row[8][status]=&row[8][ally_id]=500475&row[8][ally_tag]=Odys...&row[8][debris][metal]=0&row[8][debris][cristal]=0&row[8][activity]=0&row[8][activityMoon]=-1&galaxy=4&system=394&type=system&og_lang=es to https://docker.hispa-net.com/ogspy//mod/xtense/xtense.php from https://s159-es.ogame.gameforge.com
utilities.js:63 [1554582714529] Xtense : Status: 200
utilities.js:63 [1554582714529] Xtense : Response: {"type":"plugin univers","status":0}
utilities.js:63 [1554582714530] Xtense : setStatus : [undefined ms][OGSpy] The Xtense module configuration does not corresponds to your game server. Please check the link to the ogame server in the Xtense Module. 
```

**Comment by juliogonzalez** [comment]
And now I just noticed  that the password for the admin is leaked, so I just changed it :-P

**Comment by darknoon29** [comment]
How many characters for your password ?

**Comment by juliogonzalez** [comment]
20, and I am starting to think that is the problem.

In theory the maximum length should be 15, but it was not enforced when I installed OGspy.

Let me try to changing the password.

**Comment by darknoon29** [comment]
In your log its not the same error message
Its seems you have not configuref the right ogame server in the xtense configuration (ogspy side)

**Comment by juliogonzalez** [comment]
I did configure it.

The problem seems to be the password. I was unable to change it using OGSpy WebUI, so I did it at the MySQL database directly.

Now everything works perfect.

I will see if I can submit a patch for OGSpy. If the password is greather than 15 characters the install scripts should reject it.

This bug can be closed :-)

**Comment by darknoon29** [comment]
We have probably already fixed it for the ogspy 3.3.5 planned very soon I let you check in the release branch ;-)
